index.js

```diff
@@ -1,21 +1,21 @@
-// Inspired by (but complete rewrite of) requirejs-metagen
-let _ = require('lodash/fp')
+// Inspired by (but complete rewrite of) requirejs-metagen.
+let _       = require('lodash/fp')
```

Comment:
While we generally like doing the alignment thing, I think the standardjs style we're beginning to adopt doesn't like it :(